### DISCUSSION TITLE
Deployment to pages via `actions/deploy-pages@v2` fails - file permissions

### DISCUSSION THREAD

**Discussion by nuke-web3** [community/community#66706] [Actions] [answered]
### Select Topic Area

Bug

### Body

I updated my pages workflow in a section _not_ related to the deployment, but deployment is failing without much useful info, as I didn't change any permissions / file things and the other parts of the job are working just fine...

- changes: https://github.com/Polkadot-Blockchain-Academy/pba-book/pull/2/files#diff-267452bb53d388a7b14551d9fc53a797edc124dc9085ecc79f5339277a1e53b5
- Error: https://github.com/Polkadot-Blockchain-Academy/pba-book/actions/runs/6138776539/job/16656006592#step:2:51

Is GH having some issues...? https://stackoverflow.com/questions/71511767/github-pages-fails-to-deploy this doesn't seem to be the case...

Lower chance of a bug upstream: https://github.com/actions/deploy-pages/issues/222

**Comment by JamesMGreene** [comment, marked as answer]
@NukeManDan self-answered this one over in the adjacent issue:
- https://github.com/actions/deploy-pages/issues/222
